index.html

```diff
@@ -0,0 +1,252 @@
+            </li>
+            <li>
+                <a href="# "><img src="images/Linkedin.svg " alt="Linkedin "></a>
+            </li>
+            <li>
+                <a href="# "><img src="images/Medium.svg " alt="Medium "></a>
+            </li>
+            <li>
+                <a href="# "><img src="images/GitHub.svg " alt="GitHub "></a>
+            </li>
+            <li>
+                <a href="# "><img src="images/Angel list.svg " alt="ANGEL LIST "></a>
+            </li>
+        </ul>
+        <div>
+            <button type="button" class="bluebutton">Get my resume</button>
+        </div>
+
+        <div>
+            <ul class="listed">
+                <h3 class="codeused ">
+                    Languages
+                    <img class="down_arrow_icon " src="./images/Icon - Chevron - Down - Larger.png " alt="down " />
+                </h3>
+                <div>
+                    <li class="grey_area ">
+                        <img src="images/javascriptLogo.png " alt="Javascript " /><span>Javascript</span>
+                    </li>
+
+                    <li class="grey_area">
+                        <img src="images/html logo.png " alt="HTML " /><span>HTML</span>
+                    </li>
+                    <li class="grey_area ">
+                        <img src="images/css logo.png " alt="CSS " />
+                        <span>CSS</span>
+                    </li>
+                </div>
+            </ul>
+            <div>
+                <ul class="listed">
+                    <h3 class="codeused  ">
+                        Frameworks
+                        <img class="arrow_icon " src="images/Icon - Chevron - Right - Larger.png " alt="down " />
+                    </h3>
+
+                    <img src="images/Separator Bottom.svg " />
+                    <h3 class="codeused  "> Skills
+                        <img class="arrow_icon " src="images/Icon - Chevron - Right - Larger.png " alt="down " />
+                    </h3>
+                    <img src="images/Separator Bottom.svg " />
+                </ul>
+            </div>
+        </div>
+    </section>
+
+    <footer class="contact-menu">
+
+        <h2 class="background">Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form class="contact-form" action="https://formspree.io/f/mwkzgqen" method="POST">
+            <input type="text" id="name" name="text" placeholder="Yeremias" size="30" required>
+            <input type="email" id="email" placeholder="natajayanj14@gmail.com" size="30" required>
+            <input type="message" id="message" name="message" placeholder="Write your message here" required minlength="4" maxlength="500" size="100">
+            <div>
+                <button type="submit" class="bluebutton2">Get In touch</button>
+            </div>
+        </form>
```

Comment:
- According to the project requirements, please align the content of this section with the `flexbox` display property, please fix this 😄